crypto/bio/bss_acpt.c

```diff
@@ -226,10 +226,9 @@ static int acpt_state(BIO *b, BIO_ACCEPT *c)
                              BIO_ADDRINFO_socktype(c->addr_iter),
                              BIO_ADDRINFO_protocol(c->addr_iter), 0);
             if (ret == (int)INVALID_SOCKET) {
-                FUNCerr("socket", get_last_socket_error());
-                ERR_add_error_data(4,
-                                   "hostname=", c->param_addr,
-                                   " service=", c->param_serv);
+                ERR_raise_data(ERR_LIB_SYS, get_last_socket_error(),
+                               "calling socket hostname=%s service=%s",
```

Comment:
How about "calling socket(); hostname=%s service=%s" or "calling socket() with hostname=%s service=%s"?

Reply:
I missed that one, it should look like the parameters:
```
                 ERR_raise_data(ERR_LIB_SYS, get_last_socket_error(),
                               "calling socket(\"%s\", \"%s\")",
                                 c->param_addr, c->param_serv);
```
As was done elsewhere.

I wanted to make all the errors look like ```calling foo()``` or ```calling bar(asdf, asdf)```

Reply:
I would suggest not having the parameters look like strings, though, since they're not in reality.

Reply:
Some of them (including your own loader_file :) had them, some didn't.  I removed all the quotes.